packages.bzl

```diff
@@ -1,7 +1,7 @@
 # Each individual package uses a placeholder for the version of Angular to ensure they're
 # all in-sync. This map is passed to each ng_package rule to stamp out the appropriate
 # version for the placeholders.
-ANGULAR_PACKAGE_VERSION = "^12.0.0 || ^13.0.0-0"
+ANGULAR_PACKAGE_VERSION = "^12.1.0 || ^13.0.0-0"
```

Comment:
Sorry for the conflicting info here, I think that based on the discussion with @devversion, we should actually leave this one as `^12.0.0` since the TS 4.3-specific syntax we're using won't affect TS 4.2 users.